portfolio/src/main/webapp/style.css

```diff
@@ -115,14 +116,24 @@ body {
   list-style: none;
 }
 
+/* Styling the Comments Section */
+#commentsTextArea {
+  border-radius: 5px;
+  display: inline-block;
+  font-family: 'Roboto', sans-serif;
+  font-size: 1em;
+  padding: 5px;
+  width: 90%;
+}
+
 /* Styling the slideshow */
 .slideShowNavigation {
   background-color: white;
   color: white;
   width: 400px;
 }
 
-button {
+button, #commentsSubmitButton {
```

Comment:
alternatively, you could remove the ID and have this style apply to all buttons with

```css
button, input[type="button"], input["type=submit"] {
  /* ... */
}
```